test/e2e/space_test.go

```diff
@@ -171,7 +172,17 @@ func TestRetargetSpace(t *testing.T) {
 		err := hostAwait.CreateWithCleanup(context.TODO(), space)
 		require.NoError(t, err)
 		// wait until Space has been provisioned on member-1
-		space = VerifyResourcesProvisionedForSpaceWithTier(t, awaitilities, member1Await, space.Name, "base")
+		VerifyResourcesProvisionedForSpaceWithTier(t, awaitilities, member1Await, space.Name, "base")
```

Comment:
Shouldn't `VerifyResourcesProvisionedForSpaceWithTier()` return the latest copy of the space instead?

Reply:
It could. I thought about that but it's not really necessary for `VerifyResourcesProvisionedForSpaceWithTier` itself. If we're not doing an update after calling `VerifyResourcesProvisionedForSpaceWithTier` then the extra get would not be useful. So I thought it made more sense to do a get before update instead.